### PR TITLE
Load Config from Viper on Before hook

### PR DESCRIPTION
**Pull request by refs**
Analogous PR: https://github.com/owncloud/ocis-phoenix/pull/57

### CODE REVIEW

File: pkg/command/root.go [hunk 1]
```diff
@@ -42,3 +45,57 @@ func Execute() error {
 
 	return app.Run(os.Args)
 }
+
+// NewLogger initializes a service-specific logger instance.
+func NewLogger(cfg *config.Config) log.Logger {
+	return log.NewLogger(
+		log.Name("phoenix"),
```
Comment (micbar) [inline pkg/command/root.go]: phoenix?

File: pkg/command/root.go [hunk 2]
```diff
@@ -42,3 +45,57 @@ func Execute() error {
...
+	)
+}
+
+// ParseConfig loads proxy configuration from Viper known paths.
+func ParseConfig(c *cli.Context, cfg *config.Config) error {
+	// ParseConfig loads proxy configuration from Viper known paths.
+	logger := NewLogger(cfg)
+
+	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+	viper.SetEnvPrefix("OCIS_ACCOUNTS")
```
Comment (micbar) [inline pkg/command/root.go]: How is our naming scheme?
in proxy we have `PROXY_HTTP_ADDR`

Could we avoid naming collisions with other projects?

Comment (refs) [reply pkg/command/root.go]: I'd prefix it with `ACCOUNTS`

Comment (refs) [reply pkg/command/root.go]: by prefixing the extension name we avoid collisions

**Pull request merged**